packages/angular/src/lib/flag.directive.ts

```diff
@@ -1,39 +1,58 @@
-import {
-  Directive,
-  Input,
-  OnInit,
-  TemplateRef,
-  ViewContainerRef,
-} from "@angular/core";
-
+import { Directive, Input, OnDestroy, OnInit, TemplateRef, ViewContainerRef } from "@angular/core";
+import { Subject, takeUntil, map, distinctUntilChanged, ReplaySubject, switchMap } from "rxjs";
 import { AbbyService } from "./abby.service";
 
 @Directive({
   selector: "[featureFlag]",
```

Comment:
shouldnt there be some kind of abby branding here too? its `abbyTest` but only `featureFlag`? consistency, this is a library! :)